app/routes/_auth.signup.tsx

```diff
@@ -14,32 +17,56 @@ export async function action({ request }: ActionArgs) {
   const password = form.get('password');
   const confirmPassword = form.get('confirmPassword');
 
```

Comment:
NIT: Probably, we can keep it shorter.  
```javascript
const signUpFormData = Object.fromEntries(form);
```

Reply:
Good idea! I've made this change and updated the PR. Definitely more concise this way!